Modules/ThirdParty/DICOMParser/src/DICOMParser/Copyright.txt

```diff
@@ -3,8 +3,6 @@
   Program:   DICOMParser
   Module:    Copyright.txt
   Language:  C++
-  Date:      $Date$
```

Comment:
Isn't this a CVS thing? Subversion did not require this. Relates to commit message.